components/usage-api/usage/v1/usage.proto

```diff
@@ -15,6 +15,66 @@ service UsageService {
 
     // GetCostCenter retrieves the spending limit with its associated attributionID
     rpc GetCostCenter(GetCostCenterRequest) returns (GetCostCenterResponse) {}
+
+    // Triggers reconciliation of usage with ledger implementation.
+    rpc ReconcileUsageWithLedger(ReconcileUsageWithLedgerRequest) returns (ReconcileUsageWithLedgerResponse) {}
+
+    // Creates, or updates, a Usage record
+    rpc UpsertUsage(UpsertUsageRequest) returns (UpsertUsageResponse) {}
```

Comment:
Why would you like to expose this on API level? I thought that would be something only the recociler does.

Reply:
To be able to issue a refund for example. Ideally, humans shouldn't really touch the DB and we should go through the RPC. Having this RPC allows us to then expose a UI where you can issue a refund from the admin interface.

Reply:
Additionally, the `ReconcileUsageWithLedger` would actually call this RPC to update individual records.

Reply:
Since we have introduce an ID now and only want to update drafts but never non-drafts we should be explicit about insert (with onConflict do nothing) and updates.

Reply:
I've dropped this RPC for now.